hack/docs/compatibilitymetrix_gen_docs.go

```diff
@@ -0,0 +1,58 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/aws/karpenter/tools/kompat/pkg/kompat"
+)
+
+func main() {
+	outputFileName := os.Args[1]
+	mdFile, err := os.ReadFile(outputFileName)
```

Comment:
I wonder if it's possible to embed a separate file using Hugo/Netlify rather than having to rely on the existence of the current file and the structure of that file to inject this data.

I'm thinking that it's fairly likely that templating generation tools like that generally have a way to do something like `<-- embed fileName="my custom file" -->` and that would inject the details of that file into the document as plaintext just like it actually existed in the markdown file

Reply:
I looked into see if Hugo/Netlify supports this feature, and I was not able to find anything on it.